web/subscribers/index.php

```diff
@@ -24,4 +24,10 @@
 header('Content-type: image/svg+xml');
 
 // Get response from the URL and output its contents
-echo curl_get_contents($url);
\ No newline at end of file
+$response = (curl_get_contents($url));
+
+//Adding Commas 
+preg_match_all('!\d+!',strip_tags($response),$matches);
+$response =  str_replace($matches[0][0],number_format($matches[0][0]),$response);
```

Comment:
Improvements to code formatting:

* the parentheses around the curl_get_contents function are not necessary
* add spaces after commas separating the function parameters